server/tests/util/test_cvr_snapshot_parse.py

```diff
@@ -0,0 +1,91 @@
+from itertools import repeat, cycle
+from server.util.cvr_snapshot_parse import read_cvr_snapshots
+from server.worker.tasks import UserError
+
+
+def test_read_cvr_snapshot_one_entry():
+    entries = list(read_cvr_snapshots(["file.csv"], [repeat("test", 5)]))
+    assert entries == [("file.csv", "test")] * 5
+
+
+def test_read_cvr_snapshot_two_entries_equal_length():
+    entries = list(
+        read_cvr_snapshots(["file.csv"], [repeat("test", 3), repeat("test", 3)])
```

Comment:
For clarity, I might still include two file names in the input since we generally expect the number of file names passed to equal the number of iterators passed:

```
read_cvr_snapshots(["file1.csv", "file2.csv"], [repeat("test", 3), repeat("test", 3)])
```